action.yml

```diff
@@ -34,7 +37,7 @@ runs:
     - name: Set base and head SHAs used for nx affected
       id: setSHAs
       shell: bash
-      run: node $GITHUB_ACTION_PATH/dist/index.js ${{ github.token }} ${{ inputs.main-branch-name }} ${{ inputs.error-on-no-successful-workflow }} ${{ inputs.last-successful-event }} ${{ inputs.workflow-id }}
+      run: node $GITHUB_ACTION_PATH/dist/index.js ${{ github.token }} ${{ inputs.main-branch-name }} ${{ inputs.error-on-no-successful-workflow }} ${{ inputs.last-successful-event }} ${{ inputs.workflow-id }} ${{ inputs.working-directory }}
```

Comment:
Since `working-directory` has a default value and `workflow-id` doesn't the implementation would confuse the working directory for workflow. Move this property before the `workflow-id`

Reply:
Good call! I'll change that and update the documentation in README.md.